test/envs/regional-dr-argocd.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Environment for testing Regional-DR.
```

Comment:
Needs update.

Adding a minimal environment for argocd is a great! it makes it much easier to create the addon and debug issues later, for example upgrading to next version.

But this environment name is confusing - how is this related to regional dr?

For working on argocd deployment, do we need more than one cluster? If one cluster is enough, lets have minimal environment with one cluster. If we need more than one cluster, lets have 2 clusters (like rook.yaml). Lets use 3 clusters only if they are really needed (like ocm.yaml).